assets/js/components/user-input/UserInputPreview.js

```diff
@@ -250,11 +203,7 @@ export default function UserInputPreview( props ) {
 
 UserInputPreview.propTypes = {
 	submitChanges: PropTypes.func,
-	noFooter: PropTypes.bool,
 	goBack: PropTypes.func,
-	redirectURL: PropTypes.string,
-	errors: PropTypes.object,
-	noHeader: PropTypes.bool,
+	error: PropTypes.object,
```

Comment:
This error is not the error about answers, but the potential error for `saveUserInputSettings` action which is passed from the `UserInputQuestionnaire`.